www/src/pages/headless/combobox.md

```diff
@@ -0,0 +1,432 @@
+---
+title: Combobox
+description: "A robust component for selecting a single item from a list of options with suggestions as you type, 
+keyboard navigation, and more."
+layout: layouts/docs.njk
+permalink: /headless/combobox/
+eleventyNavigation:
+  key: combobox
+  title: Combobox
+  parent: headless
+  order: 11
+teaser: true
+demoHash: combobox
+demoHeight: 25rem
+---
+
+## Basic Example
+
+A combo box is created by the factory function `fun <T> combobox()`. `T` is the data type of the choices to be offered,
+such as a domain data type like movie characters.
+
+When the input created via `comboboxInput` is focused, a dropdown with suggestions is shown and updated as you
+type. When focused, the input shows the current input. Otherwise, the currently selected item is displayed.
+
+It is mandatory to specify a data stream or a store of type `T` as data binding via the `value` property. The component
+supports two-way data binding, i.e. it reflects a selected element from the outside by a `Flow<T>`
+but also emits the updated selection to the outside via a `Handler`.
+
+You can navigate within the selection list using the keyboard. By [[Enter]], [[Space]] or a mouse click an item is
+selected. If the combo box input loses focus or the user clicks outside the selection list, the dropdown is hidden.
+
+The selection dropdown is populated via the `comboboxItems` brick. Within the scope of this function, individual items
+can be rendered via the `comboboxItem` brick.  
+The combo box constantly evaluates which items to show based on the user's query. The results are emitted to the
```

Comment:
Before those sentences, I would suggest to explictely make clear, that not all items are renderd, but only a small amount:

"As typical use cases may offer thousands of items to choose from, the component reduces and filters those in order to support the visual recognition of a user down to a feasible size, which can be configued by `maximumDisplayedItems` . That is why the combo box constantly..."